frontend/pages/faq.tsx

```diff
@@ -16,7 +16,7 @@ const useStyles = makeStyles((theme) => {
   return {
     headerImageContainer: {
       marginBottom: theme.spacing(3),
-      [theme.breakpoints.down("md")]: {
+      [theme.breakpoints.down("lg")]: {
```

Comment:
Why did you change breakpoints from "md" to "lg"? There are other instances of this so wonder about the reasoning.